### PR TITLE
Add steps for discovering resources to README file

### PR DESCRIPTION
**Pull request by wanghongjuan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,14 @@ You will need:
  grunt platform:add:android
  grunt run
  ```
+
+
+# How to discover resources
+
+Please refer to [here] about detail steps.
```
Comment (siovene) [inline README.md]: Please refer to [here] to learn about detailed steps.

Comment (wanghongjuan) [reply README.md]: This description sounds nice, I will update it. Thanks.

**Comment by wanghongjuan** [comment]
Hi @siovene I have updated the description, could you please take a look?

File: README.md [hunk 2]
```diff
@@ -22,3 +22,14 @@ You will need:
...
  ```
+
+
+# How to discover resources
+
+Please refer to [here] to learn about detailed steps.
+
+This demo currently supports the following sensors : `Button, Buzzer, Mini Fan, Gas(MQ2), Ambient Light, Led, Motion, RGB Led, Solor, Switch(P) and Temperature`.
+
+After discover the resource, this demo can control the resource and get the expected result.
```
Comment (siovene) [inline README.md]: More like: "After the discovery process, this demo can read and control the status of the discovered resources."

Comment (wanghongjuan) [reply README.md]: Updated the description. Thanks

**Comment by siovene** [comment]
Thanks!

**Pull request merged**